File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/CQS/AllowedFunction.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.Contracts.CQS
Imports Nethereum.ABI.FunctionEncoding.Attributes
Imports StandardToken.MyContractName.DTOs
Namespace StandardToken.MyContractName.CQS

    <[Function]("allowed", "uint256")>
    Public Class AllowedFunction
        Inherits ContractMessage
    
        <[Parameter]("address", "", 1)>
        Public Property ReturnValue1 As String
        <[Parameter]("address", "", 2)>
        Public Property ReturnValue2 As String
    
    End Class


End Namespace

File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/CQS/AllowedFunction.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.Contracts.CQS;
using Nethereum.ABI.FunctionEncoding.Attributes;
using StandardToken.MyContractName.DTOs;
namespace StandardToken.MyContractName.CQS
{
    [Function("allowed", "uint256")]
    public class AllowedFunction:ContractMessage
    {
        [Parameter("address", "", 1)]
        public string B {get; set;}
        [Parameter("address", "", 2)]
        public string C {get; set;}
    }
}
